pkg/host/internal/sriov/sriov.go

```diff
@@ -258,8 +258,8 @@ func (s *sriov) DiscoverSriovDevices(storeManager store.ManagerInterface) ([]sri
 			iface.Name = name
 			iface.Mac = s.networkHelper.GetNetDevMac(name)
 			iface.LinkSpeed = s.networkHelper.GetNetDevLinkSpeed(name)
+			iface.LinkType = s.GetLinkType(name)
```

Comment:
we can get mtu, mac, link type from netlink Link obj and avoid a bunch of file operations.

not related to this PR.  i will submit a small refactor to DiscoverSriovDevices.